src/GovUk.Education.ExploreEducationStatistics.Content.Services/RedirectsService.cs

```diff
@@ -64,8 +64,18 @@ public async Task<Either<ActionResult, RedirectsViewModel>> List()
             .Distinct()
             .ToList();
 
+        var releaseRedirectViewModels = await _contentDbContext.ReleaseRedirects
+            .Where(rr => rr.Slug != rr.Release.Slug) // don't use redirects to the current live slug
+            .Distinct()
+            .Select(rr => new RedirectViewModel(
+                rr.Slug,
+                rr.Release.Slug
```

Comment:
I think labelling the attributes `fromSlug:` and `toSlug:` would help readability here.

Reply:
Unfortunately I get a `CS0853: An expression tree may not contain a named argument specification` error :(